packages/types/src/files.ts

```diff
@@ -21,3 +23,7 @@ export type File = {
 	tags: string | null;
 	metadata: Record<string, any> | null;
 };
+
+export type FileStream = {
```

Comment:
Does busboy not export this by default? Odd!

Reply:
NO! Drove me crazy. This is not exported from busboy or the busboy types from what I can find.

Reply:
busboy seems to expect it to be of type `Readable` but this does not have a truncate property. Looks like the types are outdated.

Reply:
busboy seems to expect it to be of type `Readable` but this does not have a truncate property. Looks like the types are outdated.